code/game/objects/structures/watercloset.dm

```diff
@@ -440,8 +443,8 @@
 			else
 				B.deductcharge(1)
 			user.visible_message( \
-				"[user] was stunned by his wet [O].", \
-				"<span class='warning'>You have wet \the [O], it shocks you!</span>")
+				"<span class='warning'>[user] was stunned by \his wet [O.name]!</span>", \
```

Comment:
It fucks with some macros I've noticed, most noticeably \his will turn into "its" if you use it after [O] but works fine after [O.name].
But you're right, in this case it's the same.